Packs/Zoom/Integrations/Zoom/README.md

```diff
@@ -63,40 +64,61 @@ In the Team Chat Subscription section under BOT endpoint URL, add:
    | Scope Type | Scope Name | 
    | --- |  --- | 
    | Team Chat |  Enable Chatbot within Zoom Team Chat Client /imchat:bot |
-   | Team Chat |  Send a team chat message to a Zoom Team Chat user or channel on behalf of a Chatbot /imchat:write:admin |
-   | Team Chat |  View and manage all users' team chat channels /chat_channel:write:admin |
-   | User |   View all user information /user:read:admin |
+   | Team Chat |  View a user's chat message /team_chat:read:user_message:admin |
+   | Team Chat |  View a user's chat sessions /team_chat:read:list_user_sessions:admin |
+   | Team Chat |  View a user's chat messages /team_chat:read:list_user_messages:admin |
+   | Team Chat |  Create a chat channel for a user /team_chat:write:user_channel:admin |
+   | Team Chat |  Add members to a chat channel /team_chat:write:members:admin |
+   | Team Chat |  Update a user's chat channel /team_chat:update:user_channel:admin |
+   | User |   View all user information /user:read:user:admin |
```

Comment:
I think this is fine, but I think it would be even simpler and clearer to only include the scope name (the part after the `/`) in the "Scope Name" column, with a note at the top of the table to search for the scope name in Zoom in the search bar at the top. Otherwise it is confusing and difficult to find the correct scope, since there are many similarly named ones.